meetup/services/meetup_info_fetch.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+import time
+import pdb
+
+class MeetupInfoFetch():
+  def __init__(self, meetups): 
+    self.meetups = meetups
+
+  def fat_meetups(self):
```

Comment:
This name is intriguing, but not intention revealing. What the heck are fat meetups?